### PR TITLE
Added Abhishek Kumar Card

### PR DESCRIPTION
**Pull request by abhishek44priv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">about you</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://github.com/abhishek44priv" target="_blank" title="First Resource">Github</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Email</a>
```
Comment (confused-Techie) [inline index.html:133]: You'll want to make sure to change `<a href="#"` to direct to your second 'Useful Dev Resource', I saw you put 'Email' Maybe you wanted to change that to `<a href="https://mail.google.com"` or something along those lines?

File: index.html [hunk 2]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">StackOverflow</a>
```
Comment (confused-Techie) [inline index.html:136]: Like my other comment you'll want to make sure this third 'Useful Dev Resource' points to the link you want. Likely in this case something like `<a href="https://stackoverflow.com"`.

**Review by confused-Techie — CHANGES_REQUESTED** [review]
This is looking fantastic!

I've just pointed out some things that should be addressed before this gets merged into main.

But as always thanks for your contribution!

File: index.html [hunk 3]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
...
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *Abhishek Kuamar* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Abhishek Kumar</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank"></a>
+          </p>
+          <p class="about">about you</p>
```
Comment (confused-Techie) [inline index.html:125]: Also almost forgot, this "about you" section should tell us a little bit about yourself, feel free to refer to the [documentation](https://github.com/Syknapse/Contribute-To-This-Project#step-4-open-the-indexhtml-file) for help knowing what's expected on this section.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@abhishek44priv, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**